release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -70,6 +70,18 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
 [id="ocp-4-15-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
```

Comment:
```suggestion
[id="ocp-4-15-administrator-perspective"]
```

Reply:
Also wondering if these notes should only cover the [Customer Happiness epic](https://issues.redhat.com/browse/CONSOLE-3542)?